### ISSUE TITLE
reg loss function

### ISSUE THREAD

**Issue by wenxi2** [ChenhongyiYang/QueryDet-PyTorch#101]
Hello, have you ever tried to use GIoU regression loss to train the model? Why does eta in model training keep increasing when I use GIoU loss function? Look forward to your answer, thank you!
